### ISSUE TITLE
Access to task, milestones is only available from Projects main page?

### ISSUE THREAD

**Issue by adobrovo** [projectknife/Projectfork#1684]
Hi, I'm testing the component, I've created 4 projects, and I added in the main menu direct access to the Projects page, so I can see all 4 projects and below each projects the buttons for Task, Milestones, comments, etc. 
Then, when I access to the project, I only see the description and the option to add comments, but theres is no buttons to check the tasks o milestones. As I havent found buttons to navigate to the previous page the only way to come back to see the tasks is pressing the back button on my navigator or using the main menu option I added.
Am I missing something? Did I forget to add a module or something to allow my users to access to the diferents objects in my projects once they are inside the project detail, instead having to go back to the main page of the projects, to access to them?

**Comment by angek** [comment]
have you published and assigned the projectfork menu module?

**Comment by adobrovo** [comment]
Yes, I published and asigned the menu module But I dont want to include the menu in the page, I wanted to make it very simple for my users,  what I recommend you people is to add the same buttons you find below each project in the PROJECT section, but inside the DASHBOARD, which for what I saw is the same page you are redirected when you press over the title of the project when you are in the Project section. 
I would like to share the projects  in social networks and allow people to view the task and milestones associated to this project avoiding to teach them how to use the projectfor menu.

Is this possible?

Regards
Aldo
